### ISSUE TITLE
Virus Total detects malicious code

### ISSUE THREAD

**Issue by dodgyb** [AndyFul/ConfigureDefender#2]
Checked ConfigureDefender_x64.exe on Virus Total and 3 engines reported malicious software:

https://www.virustotal.com/#/file/16c103ee6b1350dc80b01548e6cdf63d00905f0307f7ea8d75fa6ad9e549dcc5/detection

Antiy-AVL - Trojan/Generic.ASVCS3S.1E5

CrowdStrike Falcon - malicious_confidence_60% (W)

McAfee-GW-Edition - BehavesLike.Win64.Dropper.tc

**Comment by krystian3w** [comment]
https://www.virustotal.com/#/file/9941cf56d5d8aeee9227fc8806efe3f633c3a3cb402b842052970c8fe82a8d14/detection

Antiy-AVL - Trojan/Generic.ASVCS3S.1E5

SentinelOne - static engine - malicious

Sophos ML - heuristic

**Comment by sundhaug92** [comment]
x86 - https://www.virustotal.com/nb/file/9941cf56d5d8aeee9227fc8806efe3f633c3a3cb402b842052970c8fe82a8d14/analysis/

x64 - https://www.virustotal.com/nb/file/112366df0ddc6102c5d7efe9e59ca37ff2abb03cc3b70516e6767dc0a29157af/analysis/

**Comment by AndyFul** [comment]
ConfigureDefender executables are always submitted to Microsoft, Avast, Symantec, BitDefender, and Emsisoft before publishing. The application changes Defender settings and some Windows Policies, so some behavior-based detections can flag it as suspicious/malicious.

**Comment by firebowl** [comment]
Microsoft Defender also blocks download because of a virus...

**Comment by AndyFul** [comment]
> Microsoft Defender also blocks download because of a virus...

Are you sure? Many people on BleepingComputer, MalwareTips, and WildersSecurity forums use ConfigureDefender and no one reported such an issue. I checked it also on my computer with Edge. Both SmartScreen and PUA protection in Edge allow downloading without alerts. I can also run the application without Defender's alert. Could you please recheck the isuue?

**Comment by AndyFul** [comment]
It seems that @firebowl did not download the ConfigureDefender installer but the whole GitHub repository in ZIP archive: ConfigureDefender-master.zip. For an unknown reason, the whole archived repository was flagged by Defender despite the fact that all files included in the archive were detected as clean. I submitted the archived repository to Microsoft today and the false positive has been removed with AntivirusSignatureVersion: 1.329.2695.0.
Thank @firebowl, guys from MalwareTips and Wilderssecurity forums the issue has been finally solved.
